test/uk/gov/hmrc/icedsubscriptionfrontend/views/AlreadyEnrolledPageSpec.scala

```diff
@@ -57,24 +59,41 @@ class AlreadyEnrolledPageSpec extends SpecBase {
     }
 
     "have a only one page heading" in {
-      document.select(Selectors.h1).text shouldBe "Enrol with the Safety and Security service"
+      document.select(Selectors.h1).text shouldBe "Your organisation is already enrolled with S&S GB"
```

Comment:
IIRC a heading is not the same as the browser tab title it should form part of the title.

So I think line 49 above should read:
`document.title shouldBe "Your organisation is already enrolled with S&S GB - Enrol with the Safety and Security service - GOV.UK" `